### PR TITLE
Added Networkx Tutorial python script

### PR DESCRIPTION
**Pull request by tindi-plus**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# my testing notebook. For learning.
+practice_notebook.py
```
Comment (MridulS) [inline .gitignore:132]: could you remove this change from .gitignore ? Thanks!

**Comment by MridulS** [comment]
Also it helps in review if you create new pull request for every task (you can create a new git branch for a new task) otherwise the pull request gets big and it takes more time to review it. Like here both the python script and jupyter notebook tasks are present.

**Comment by tindi-plus** [comment]
Ohhh....Okay. Thank you very much for the information.

I will do that right away.

A quick one: so should I close each pull request after creating it? Because
whenever I push, the PR is automatically updated.

**Comment by MridulS** [comment]
> A quick one: so should I close each pull request after creating it? Because
> whenever I push, the PR is automatically updated.

This is because you are using the main branch for everything. You should be creating a new branch for every new task.
to get a rough idea about the development workflow you can read through https://networkx.org/documentation/latest/developer/contribute.html#development-workflow

**Comment by tindi-plus** [comment]
I understand now. Thank you very much.

**Comment by MridulS** [comment]
Looks good! Thanks @tindi-plus!

**Pull request merged**